tests/rados/pool_tests.py

```diff
@@ -1025,7 +1038,9 @@ def run(ceph_cluster, **kw):
                     )
                     raise Exception("Reads failed on the cluster Error")
 
-                if pool_obj.do_rados_put(client=client_node, pool=pool_name, nobj=10):
+                if pool_obj.do_rados_put(
+                    client=client_node, pool=pool_name, nobj=10, obj_name="test-obj-1"
```

Comment:
By providing obj name, we are just overwriting the same object `nobj` times, are we not fixing the `do_rados_put` method in this PR?

Reply:
Fixing the method was not the intention of the PR, however, if needed, we can take that up.